spec/routing/exports_routing_spec.rb

```diff
@@ -0,0 +1,35 @@
+require "rails_helper"
```

Comment:
Glad you're writing tests. Feel free to do the routing specs if you find them helpful, but i don't really do them